Added new 'General' reply

bremea opened the pull request:
Added a dropdown box allowing user to select a subforum. Then, a message is then generated for the user to copy:

> I think topic this would fit better in the "xyz" subforum. I'll ask to have it moved for you.

bremea committed:
Added new 'General' reply

Allows user to select subforum to generate a 'should belong in x subforum' text

Accio1 reviewed (changes requested):
This is a neat idea, but I'd like to add some CSS to the dropdown, so that it doesn't just look like the default dropdown. I'm pretty busy right now, so I might not be able to add the CSS for a while. If you want to add the CSS instead, I would like the dropdown to be inbetween the textarea and the copy text button, as well as adding a custom style to the dropdown (similar to the one in the Scratch Footer which is used to select a language).

bremea commented:
Gotcha, I'll start work on it now.

bremea committed:
added <select> styles

bremea committed:
Fixed select issue

Fixed issue where "Things I'm Reading and Playing" didn't appear in selection menu.

bremea commented:
Couldn't get the dropdown next to the button, just sort of didn't look great (plus shrinking the width affected how some options displayed in the dropdown menu. Did get it to look like the scratch dropdown menu, though.
